.github/actions/devcontainer_run_command/action.yml

```diff
@@ -185,6 +185,7 @@ runs:
             && inputs.CORE_APP_SERVICE_PLAN_SKU) || 'P1v2' }}" \
           -e WORKSPACE_APP_SERVICE_PLAN_SKU="${{ (inputs.WORKSPACE_APP_SERVICE_PLAN_SKU != ''
             && inputs.WORKSPACE_APP_SERVICE_PLAN_SKU) || 'P1v2' }}" \
+          -e TF_VAR_rp_bundle_values='{"tre_test_custom_val_1": "Value 1", "tre_test_custom_val_2": "Value 2"}' \
```

Comment:
Don't you need to pass a real value here?
What this is meant to do?

Reply:
this is just passing in some arbitrary values to RP_BUNDLES - so that we can a) validate that my change doesn't prevent the RP starting, and b) highlight a future change that might break the way RP_BUNDLE values are dealt with, as this would break the build.